lib/helpers.js

```diff
@@ -0,0 +1,686 @@
+    *  @example {{#gte 5 2}}True{{/gte}}
+    */
+module.exports.gte = function(value, value2, block) {
+	if(Number(value) >= Number(value2))
+		return block.fn(this);
+	return block.inverse(this);
+};
+
+/**
+    *  Lower than block
+    *  Checks first argument against the second one.
+    *  @param {int} value
+    *  @param {int} value2
+    *  @returns If the condition is true return the content inside of block
+    *  @example {{#lt 1 2}}True{{/lte}}
+    */
+module.exports.lt = function(value, value2, block) {
+	if(Number(value) < Number(value2))
+		return block.fn(this);
+	return block.inverse(this);
+};
+
+/**
+    *  Lower equal block
+    *  Checks first argument against the second one.
+    *  @param {int} value
+    *  @param {int} value2
+    *  @returns If the condition is true return the content inside of block
+    *  @example {{#lte 1 1}}True{{/lt}}
+    */
+module.exports.lte = function(value, value2, block) {
+	if(Number(value) <= Number(value2))
+		return block.fn(this);
+	return block.inverse(this);
+};
+
+/**
+    *  For block
+    *  For loop
+    *  @param {int} from
+    *  @param {int} to
+    *  @example {{#forLoop 0 10}}{{this}}{{/for}} 0 ... 9
+    */
+module.exports.forLoop = function(from, to, block) {
+
+	let accumulator = '';
+
+	for(let i = from; i < to; i++)
+		accumulator += block.fn(i);
+
+	return accumulator;
+};
+
+/**
+    *  notIn block
+    *  Check if the value doesn't exists in haystack
+    *  @param {int} needle - The needle
+    *  @param {string} haystack - Comma separated values}
+    *  @returns If the condition is true return the content inside of block
+    *  @example {{#notIn id "245,272,300"}}I'm not 245, 272, 300{{/notIn}}
+    */
+module.exports.notIn = function(needle, haystack, block) {
+
+	haystack = haystack.toString().split(',');
+
+	if(typeof needle === 'undefined' || !(~haystack.indexOf(needle.toString())))
+		return block.fn(this);
+
+	return block.inverse(this);
+};
+
+/**
+    *  in block
+    *  Check if the needle exists in haystack
+    *  @param {int} needle - The needle
+    *  @param {string} haystack - Comma separated values
+    *  @returns If the condition is true return the content inside of block
+    *  @example {{#in id "245,1,300"}} Im {{id}} {{/in}}
+    */
+module.exports.inHelper = function(needle, haystack, block) {
+
+	haystack = haystack.toString().split(',');
+
+	if(typeof needle === 'undefined' || haystack.find(element => element === needle.toString()))
+		return block.fn(this);
+
+	return block.inverse(this);
+};
+
+/**
+    *
+    *  Checks if all the values are false
+    *  @param {mixed} args - The values to check
+    *  @return {boolean} Return the result of the condition
+    *  @example {{false value1 value2}} ==> "true" or "false"
+    */
+module.exports.falseHelper = (...args) => {
+	args.pop();
+
+	const trueValues = args.filter(element => Boolean(element) === true);
+
+	return !trueValues.length;
```

Comment:
```suggestion
	const falseValues = args.filter(element => !Boolean(element));

	return !!falseValues.length;
```

Reply:
Aca estoy queriendo hacer esto:
Busco los valores que son `truthy` ⬇️
```js
const trueValues = args.filter(element => Boolean(element) === true);
```
Si no hay retorno `true` porque significa que no hay ningun valor `truthy`⬇️
```js
return !trueValues.length;
```

Reply:
Estas haciendo cosas por la negativa.. osea lo contrario a lo que queres hacer..

Y después, en el helper contrario, haces lo mismo al réves..

Es un enredó..

Aca, este helper trata sobre si hay valores falsy.. entonces podés usar `some` y `!Boolean(element)`.. osea buscas valores falsy.. 

(en el otro helper que trata sobre si hay valores truthy, buscas valores truhy)

Reply:
Lo que trata es esto;-
_Checks if all the values are false_
Como hago para verificar que `todos` los argumentos sean falsos?

Reply:
```js
someArray.every(item => !Boolean(item)) // si todos los items son falsos devuelve true
```